bears/jinja2/Jinja2Bear.py

```diff
@@ -184,6 +190,44 @@ def check_for_variable_spacing_issues(self,
                     end_column=m.end(0) + 1,
                     diffs=diff)
 
+    def check_for_statement_spacing_issues(self,
+                                           file,
+                                           filename,
+                                           line,
+                                           line_number,
+                                           statement_spacing):
+        """
+        Checks any statement in the given line for spacing issues.
+
+        :param file:
+            The content of the file currently being inspected.
+        :param filename:
+            The name of the file currently being inspected.
+        :param line:
+            The content of the line currently being inspected.
+        :param line_number:
+            The current line number.
+        :param statement_spacing:
+            The number of spaces required on each side of a statement tag.
+        """
+        for match in self.STATEMENT_REGEX.finditer(line):
+            content = match.group('content')
+            if not has_required_spacing(content, statement_spacing):
+                diff = generate_spacing_diff(
+                    file, filename, line, line_number, match,
+                    statement_spacing)
+                yield Result.from_values(
+                    origin=self,
+                    message='Statement blocks not spaces with '
+                            '`{}` spaces on each side.'.format(
+                                statement_spacing),
+                    file=filename,
+                    line=line_number,
+                    column=match.start() + 1,
+                    end_line=line_number,
+                    end_column=match.end() + 1,
+                    diffs=diff)
```

Comment:
>         :param diffs:
>            A dictionary with filename as key and ``Diff`` object
>            associated with it as value.

Reply:
Nvm, the variable name is just a bit misleading :) But it's okay, as the existing code calls it the same.

Reply:
`diff` is not a parameter for the function I have written here. It is being passed to Result class. Should i still define this as well?

Reply:
No no keep it as is, `generate_spacing_diff` wraps everything properly inside a dict like required :)